velox/dwio/common/Options.h

```diff
@@ -606,6 +606,7 @@ struct WriterOptions {
   std::optional<bool> orcWriterStringDictionaryEncodingEnabled{std::nullopt};
   std::map<std::string, std::string> serdeParameters;
   std::optional<uint8_t> parquetWriteTimestampUnit;
+  std::optional<std::string> parquetWriteTimestampTimeZone;
```

Comment:
CC: @pedroerp

Reply:
@JkSelf I'm moving the parquet-specific options to the parquet library in this PR:

https://github.com/facebookincubator/velox/pull/10470

I'll try to get it merged today or early next week in the latest. Could you wait and rebase on top of that? It should be cleaner to add the option after that refactor.

Reply:
@pedroerp Updated based on https://github.com/facebookincubator/velox/pull/10470